stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -7,29 +7,34 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-
-    <FrameLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/bottom_sheet"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
         android:clipToPadding="true"
         android:background="?colorSurface"
         app:layout_behavior="com.google.android.material.bottomsheet.BottomSheetBehavior">
 
+
+
         <androidx.fragment.app.FragmentContainerView
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.0"
+            app:layout_constraintBottom_toTopOf="@+id/buy_button"
             android:id="@+id/fragment_container"
             android:layout_width="match_parent"
-            android:layout_height="match_parent" />
-    </FrameLayout>
+            android:layout_height="wrap_content" />
 
-    <Button
-        android:id="@+id/buy_button"
-        android:layout_margin="@dimen/stripe_paymentsheet_outer_spacing"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:enabled="false"
-        android:text="Pay $25.00"
-        app:layout_anchor="@+id/bottom_sheet"
-        app:layout_anchorGravity="bottom" />
+        <Button
+            android:id="@+id/buy_button"
+            android:layout_margin="@dimen/stripe_paymentsheet_outer_spacing"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:enabled="false"
+            android:text="Pay $25.00"
```

Comment:
string resource?

Reply:
This is just a placeholder. Will be replaced soon